static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -212,14 +217,31 @@ function TaskGroup({
   expanded,
   hidePanel,
   toggleable = true,
+  group,
 }: TaskGroupProps) {
+  const organization = useOrganization();
   const [isExpanded, setIsExpanded] = useState(expanded);
   const {completedTasks, incompletedTasks} = groupTasksByCompletion(tasks);
+  const previousCompletedTasksCount = usePrevious(completedTasks.length);
 
   useEffect(() => {
     setIsExpanded(expanded);
   }, [expanded]);
 
+  useEffect(() => {
+    if (
+      completedTasks.length !== tasks.length ||
+      previousCompletedTasksCount !== tasks.length - 1
```

Comment:
What if multiple tasks get marked as done at once?
Instead we could check `previousCompletedTaskCount === task.length`, right?

Reply:
no it still does not solve the issue. To solve this I decided to use local storage